Pravin Chavan/CineScope/app/src/main/java/com/prawin/cinescope/MainActivity.kt

```diff
@@ -0,0 +1,59 @@
+package com.prawin.cinescope
+
+import android.os.Bundle
+import androidx.activity.ComponentActivity
+import androidx.activity.compose.setContent
+import androidx.activity.enableEdgeToEdge
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.prawin.cinescope.ui.theme.CineScopeTheme
+import com.prawin.cinescope.ui.theme.reddish
+import com.prawin.cinescope.ui.theme.white
+
+class MainActivity : ComponentActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        enableEdgeToEdge()
+        setContent {
+            CineScopeTheme {
+                Scaffold(modifier = Modifier.fillMaxSize()) { innerPadding ->
+                    splashScreen(innerPadding)
+                }
+            }
+        }
+    }
+}
+
+@Composable
+private fun splashScreen(innerPadding: PaddingValues) {
```

Comment:
Why are we keeping the compose function in this file? can we have a separate file for this?